CONTRIBUTING.md

```diff
@@ -9,7 +9,10 @@ This file is not comprehensive yet and might still contain mistakes or
 unclear indications, please consider contributing.
 
 ## Proof style
+
 ### General guidelines
+- One important guideline is to structure proofs in blocks, i.e.,
```

Comment:
Maybe an example would help here to clarify this for newcomers?

Reply:
I am not sure but let my add a link for the sake of discussion.

Reply:
@affeldt-aist Thank you!